Composer/packages/extension/src/manager/manager.ts

```diff
@@ -245,12 +245,15 @@ class ExtensionManager {
       const extensionInstallPath = path.dirname(fullPath);
       const packageJson = (await readJson(fullPath)) as PackageJSON;
       const isEnabled = packageJson?.composer && packageJson.composer.enabled !== false;
+      const metadata = getExtensionMetadata(extensionInstallPath, packageJson);
```

Comment:
why was this moved out of the `if` statement if the `else if` does not contain any reference to `metadata`?

Reply:
Good question. I had a version where I was just disabling the extension but setting the other metadata fields. I felt like removing the extension is better but forgot to move this back.

Reply:
we can just move it back on some other PR